### DISCUSSION TITLE
Convert Older Habits into new 'Yes/No' or 'Measurable' Format.

### DISCUSSION THREAD

**Discussion by alphanerdgen** [iSoron/uhabits#934] [Feature Requests]
Been using your app since 2-3 years. Would be great if we could convert older Habits into new formats, if possible. Can't manually copy the data as that'd take way too much time.

**Comment by ConorSheehan1** [comment]
@alphanerdgen I made a little python script to do this https://github.com/ConorSheehan1/uhabits_converter
